### PR TITLE
Adding coverage for exceptions.py

### PR DESCRIPTION
**Pull request by shaneallcroft**
Working on unchecked exceptions.

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -142,3 +142,8 @@ def test_url_to_repo_org(self):
         owner, repo = url_to_repo_org('https://github.com/allenh1/p2os')
         self.assertEqual(owner, 'allenh1')
         self.assertEqual(repo, 'p2os')
+
+    def test_unknown_platform(self):
+        """Test resolve_dep with bad OS"""
+        with self.assertRaises(UnknownPlatform):
```
Comment (allenh1) [inline tests/test_utils.py]: I think you need to add an import for `superflore.exceptions.UnknownPlatform`

File: tests/test_utils.py [hunk 2]
```diff
@@ -142,3 +142,8 @@ def test_url_to_repo_org(self):
...
+            ret = rsolve_dep('cmake', 'Windoughs8')
```
Comment (allenh1) [inline tests/test_utils.py]: `s/rsovle_dep/resolve_dep`

**Comment by tfoote** [comment]
@shaneallcroft You might find it helpful to run the unit tests yourself locally before opening a pull request. You can see how the CI is running them by looking at the [`.travis.yml`](https://github.com/ros-infrastructure/superflore/blob/master/.travis.yml)

**Comment by shaneallcroft** [comment]
@tfoote Will do! Sorry for the noise.

**Comment by allenh1** [comment]
@shaneallcroft Thanks for adding that missing test! Unfortunately, the others in `exceptions.py` are not quite doable for the moment, since the test files are missing entirely. Next on my list, however, is to start the `repo_instance.py` tests.

So, pending @tfoote's opinion, I think this can be merged.

**Pull request merged**